doc/offline-signing-tutorial.md

```diff
@@ -0,0 +1,259 @@
+    "amount": 0.00090000,
+    "confirmations": 1,
+    "spendable": true,
+    "solvable": true,
+    "desc": "wpkh([413037c1/84h/1h/0h/0/0]02b66303a69d660e98c49320afc28dd9abd8f34c9d59c4ca2148a243fa811ea6c8)#6d9cgsep",
+    "parent_descs": [
+      "wpkh([413037c1/84h/1h/0h]tpubDCjzsbgeDYJj2Xgpb5T5CjQ97muocfoQDkYu68RJvu4sspunz82G7pBfFcWUvCGj4AhQoDkbX24jxXyrREyVUAba6B8qKGkFNPyA8GBQ3kA/0/*)#3us38eqx"
+    ],
+    "safe": true
+  }
+]
+```
+
+## Step 2: Sending money from the offline wallet
+
+### Create an Online Receiving Wallet
+
+1. On the online machine, create an online wallet named `online_receiving_wallet`.
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -named createwallet wallet_name="online_receiving_wallet"
+{
+  "name": "online_receiving_wallet"
+}
+```
+
+2. Generate a new address on the `online_receiving_wallet` to receive funds from the offline wallet.
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -rpcwallet="online_receiving_wallet" getnewaddress
+tb1q9k5w0nhnhyeh78snpxh0t5t7c3lxdeg3erez32
+```
+
+### Create and Export a PSBT Transaction
+
+1. Create a PSBT transaction using the online `watch_only_wallet`, sending funds to the `online_receiving_wallet` address.
+- Export the psbt transaction to `funded_psbt.txt` for easy portability to the `offline_wallet`
+
+```sh
+[online]$ ./src/bitcoin-cli -signet -rpcwallet="watch_only_wallet" walletcreatefundedpsbt '[]' '[{"tb1q9k5w0nhnhyeh78snpxh0t5t7c3lxdeg3erez32": 0.0008}]' | jq -r '.psbt' >> ~/funded_psbt.txt
+[online]$ cat ~/funded_psbt.txt
+cHNidP8BAHECAAAAAWXoonGoVULy7an4Z+TcQAAuT0axtGZAcS/Qp3AQ2QgEAQAAAAD9////AoA4AQAAAAAAFgAULajnzvO5M38eEwmu9dF+xH5m5RGDJgAAAAAAABYAFMmkmFUvb9NZrPu+VxeaMLiwdbHHAAAAAAABAHECAAAAAefmKOqzoEKbtyEsrdzdiOId8oXPcNM0OBHxeNPfRdHgAAAAAAD9////AiPiDQAAAAAAFgAUZJLFRE3DQ9BMSF62gBYs69jH9A+QXwEAAAAAABYAFNGCVTKST/+XBKjmy7QD6IJZlIxDcGsCAAEBH5BfAQAAAAAAFgAU0YJVMpJP/5cEqObLtAPoglmUjEMiBgK2YwOmnWYOmMSTIK/Cjdmr2PNMnVnEyiFIokP6gR6myBhBMDfBVAAAgAEAAIAAAACAAAAAAAAAAAAAACICAhyFX/Tv0f4V3Rx7cR0zzR4AwWzGRqlZGtpRnnnQFrsrGEEwN8FUAACAAQAAgAAAAIABAAAAAAAAAAA=
+```
+
+### Decode and Analyze the PSBT Transaction
+
+1. Decode and analyze the PSBT transaction on the `offline_wallet` using the `funded_psbt.txt` file.
+
+```sh
+[offline]$ ./src/bitcoin-cli -signet decodepsbt $(cat ~/funded_psbt.txt)
+{
+    ...
+}
+[offline]$ ./src/bitcoin-cli -signet analyzepsbt $(cat ~/funded_psbt.txt)
+{
+  "inputs": [
+    {
+      "has_utxo": true,
+      "is_final": false,
+      "next": "signer",
+      "missing": {
+        "signatures": [
+          "d1825532924fff9704a8e6cbb403e88259948c43"
+        ]
+      }
+    }
+  ],
+  "estimated_vsize": 141,
+  "estimated_feerate": 0.00001000,
+  "fee": 0.00000141,
+  "next": "signer"
+}
+```
+
+### Process and Sign the PSBT Transaction
+
+1. Process and sign the PSBT transaction on the `offline_wallet`.
+
+```sh
+[offline]$ processed_psbt=$(./src/bitcoin-cli -signet -rpcwallet="offline_wallet" walletprocesspsbt $(cat ~/funded_psbt.txt) | jq -r .psbt)
```

Comment:
Hm I haven't noticed this before but `walletprocesspsbt` can finalize (default is to finalize if possible) and so if this command returned hex in addition to base64, we could omit the `finalizepsbt` step... right? Might be another good follow-up PR ;-)